rfc#0001

```diff
@@ -0,0 +1,145 @@
+# WASM binary loading into an Enarx Keep
+- Authors: [Mike Bursell](https://github.com/MikeCamel)
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2020-01-10
+- Status Note: (has been subject to initial discussion and consensus)  
+- Supersedes: n/a
+- Start Date: 2020-02-16
+- Tags: architecture, webassembly
+
+## Summary
+
+This RFC describes the architecture adopted by Enarx for loading Wasm
+packages into a prepared Keep using a secure channel. 
+
+## Motivation
+
+Once a Keep is prepared with microkernel, Wasm runtime, etc., the
```

Comment:
~~microkernel~~ shim

Microkernel is true in the VM-based keeps but not the process-based keeps. Shim is a good term that fits both.